op-challenger/game/fault/contracts/oracle_test.go

```diff
@@ -258,3 +258,171 @@ func TestMetadata_Countered(t *testing.T) {
 	meta.setCountered(false)
 	require.False(t, meta.countered())
 }
+
+func TestGetLeafBlocks(t *testing.T) {
```

Comment:
Given it's just a simple return of the error and we already have a linter that prevents us from ignoring the error I don't think we need a specific test for this.  Notably we haven't tested the error case for other methods.

If there was some particular error handling that needed to be done instead of just returning the error, then yeah definitely should test that, but if it's just a `if err != nil { return err }` kind of block the linter provides enough coverage and the extra code and test time required for a unit test isn't really justified.